tests/casper/test_view.py

```diff
@@ -167,3 +164,18 @@ def test_multiple_messages_arriving_resolve():
     validator_1.receive_messages(test_lang.network.global_view.justified_messages.values())
 
     assert len(validator_1.view.justified_messages) == 7
+
```

Comment:
Have to have this weird workaround b/c of the difference in starting messages between protocols. Might be better to move these into their respective folders, but because we can "clump" them this might be satisfactory.